### DISCUSSION TITLE
Can I break a chapter in distinct html files?

### DISCUSSION THREAD

**Discussion by jandermoreira** [quarto-dev/quarto-cli#6524] [Q&A] [answered]
### Description

I have a long chapter in a book, with lots of figures. And this results in a long html page.

It doesn't make sense to me to split this chapter into other chapters, but I would like to have separate html files with the inner sections.
For example, having 2.1, 2.2, and so on on separate pages.

It's possible?

**Comment by mcanouil** [comment, marked as answer]
You can't because what would be the "link" between those pages?
So for you to get smaller HTML chapter page would be to use parts to subdivide your long chapter or to rethink your chapter structure or use HTML features to help navigate in it.
